packages/dev/core/src/Maths/math.vector.functions.ts

```diff
@@ -29,3 +32,39 @@ export function Vector3ToFixed(vector: Vector3, decimalCount: number): string {
 export function Vector4ToFixed(vector: Vector4, decimalCount: number): string {
     return `{X: ${vector.x.toFixed(decimalCount)} Y: ${vector.y.toFixed(decimalCount)} Z: ${vector.z.toFixed(decimalCount)} W: ${vector.w.toFixed(decimalCount)}}`;
 }
+
+/**
+ * Returns the angle in radians between two quaternions
+ * @param q1 defines the first quaternion
+ * @param q2 defines the second quaternion
+ * @returns the angle in radians between the two quaternions
+ */
+export function GetAngleBetweenQuaternions(q1: DeepImmutable<Quaternion>, q2: DeepImmutable<Quaternion>): number {
+    return Math.acos(Clamp(Quaternion.Dot(q1, q2))) * 2;

```

Comment:
Is calling Quaternion static functions going to bring in the whole class?

Reply:
yup there should only be type imports from math

Reply:
I'm not sure we should fix this now. I would have to bring a bunch more functions here. We should discuss naming conventions before moving them.